inventory/pom.xml

```diff
@@ -52,6 +52,11 @@
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
```

Comment:
Why do we need this dependency ? I think we already have spring-boot-starter-test in parent pom.xml

Reply:
My mistake, I removed this dependency